nltk/tag/perceptron.py

```diff
@@ -197,7 +197,7 @@ def train(self, sentences, save_loc=None, nr_iter=5):
         # Pickle as a binary file
         if save_loc is not None:
             with open(save_loc, 'wb') as fout:
-                pickle.dump((self.model.weights, self.tagdict, self.classes), fout, -1)
+                pickle.dump((self.model.weights, self.tagdict, self.classes), fout, 2)
```

Comment:
I think this change deserves a comment, something like

    # Make pickle dump compatible with Python 2
    # https://docs.python.org/3/library/pickle.html#data-stream-format